chain/client/src/client_actor.rs

```diff
@@ -555,7 +555,11 @@ impl Handler<Status> for ClientActor {
         Ok(StatusResponse {
             version: self.client.config.version.clone(),
             protocol_version,
-            latest_protocol_version: PROTOCOL_VERSION,
+            latest_protocol_version: if self.client.config.use_next_protocol_version {
```

Comment:
This should be extracted into a separate function. Assuming I understand the way this will be used, all the new features will have `if get_protocol_version() >= SOME_CONSTANT { do new bla } else { do old bla }`, so the `get_protocol_version()` should be readily available.